utility/src/main/java/org/oppia/android/util/html/tags/BUILD.bazel

```diff
@@ -0,0 +1,19 @@
+"""
+General purpose utilities for html tags.
+"""
+
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
+    name = "tag_handler",
+    srcs = [
+        "BulletTagHandler.kt",
+        "ConceptCardTagHandler.kt",
+        "MathTagHandler.kt",
+    ],
+    visibility = ["//:oppia_api_visibility"],
```

Comment:
It seems like this library is not widely used, could you minimize the visibility for this?

Reply:
It's used by file outside this package so I made it oppia api visibility

Reply:
Done